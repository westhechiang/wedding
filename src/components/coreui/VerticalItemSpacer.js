import styled from 'styled-components';

const VerticalItemSpacer = styled.div`
  width: ${props => props.width || '100%'};
  margin-bottom: ${props => (props.space || '10px')};
`;

export default VerticalItemSpacer;
