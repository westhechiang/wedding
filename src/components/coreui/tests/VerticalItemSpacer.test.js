import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import VerticalItemSpacer from '../../../components/coreui/VerticalItemSpacer';

describe('<VerticalItemSpacer />', () => {
  it('should match its empty snapshot', () => {
    const wrapper = shallow(<VerticalItemSpacer />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });

  it('should match its snapshot when passed a "space" prop', () => {
    const wrapper = shallow(<VerticalItemSpacer space="10px" />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
