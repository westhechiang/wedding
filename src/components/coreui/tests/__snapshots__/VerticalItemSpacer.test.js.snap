// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<VerticalItemSpacer /> should match its empty snapshot 1`] = `
<div
  className="sc-bdVaJa lkyiYB"
/>
`;

exports[`<VerticalItemSpacer /> should match its snapshot when passed a "space" prop 1`] = `
<div
  className="sc-bdVaJa lkyiYB"
/>
`;
